docs/tools/local-testing/dockerized-testing.md

```diff
@@ -0,0 +1,193 @@
+# Dockerized setup
+
+## Dockerized local setup: installation & setup
+
+Let's delve into the setup process for Dockerized Local Testing Environment.
+
+### Prerequisites
+
+Make sure `Docker` and `docker-compose` are installed on your system. If not, follow the [installation guide](https://docs.docker.com/get-docker/). Familiarity with the zkSync Hardhat plugins is also recommended. If you're new to zkSync development with Hardhat, check out the [getting started section here](../../tools/hardhat/getting-started.md).
+
+### Setting up the testing environment
+
+To clone the dockerized project, use the following command:
+
+```
+git clone https://github.com/matter-labs/local-setup.git
+```
+
+### Starting the local node
+
+To launch the zkSync Era locally, run the `start.sh` script:
+
+```
+cd local-setup
+./start.sh
+```
+
+This command initiates three docker containers:
+
+- Postgres: The database for zkSync.
+- Local Geth node: The L1 for zkSync.
+- The zkSync node itself.
+
+:::tip
+ The first `start.sh` script execution should go uninterrupted. If the bootstrapping process halts unexpectedly, reset the local zkSync state and try again.
+:::
+
+### Network details
+
+By default, the HTTP JSON-RPC API is served via port `3050`, while the WebSocket (WS) API is accessed through port `3051`.
+
+**L1 RPC:** `http://localhost:8545`
+**L2 RPC:** `http://localhost:3050`
+**Network Id:** 270
+
+### Resetting the zkSync state
+
+To reset the zkSync state, run
+
+ the `./clear.sh` script:
+
+```
+./clear.sh
+```
```

Comment:
```suggestion
### Resetting the zkSync state

To reset the zkSync state, run the `./clear.sh` script:

```
./clear.sh
```
::: info Update docker images 
The `clear.sh` script will also pull the most recent Docker images so it can be used to update the dockerized local setup.
:::
```